src/java.management/share/classes/java/lang/management/DefaultPlatformMBeanProvider.java

```diff
@@ -56,8 +56,8 @@
          */
         initMBeanList.add(new PlatformComponent<ClassLoadingMXBean>() {
             private final Set<String> classLoadingInterfaceNames =
-                    Collections.unmodifiableSet(Collections.singleton(
-                            "java.lang.management.ClassLoadingMXBean"));
+                    Collections.singleton(
+                            "java.lang.management.ClassLoadingMXBean");
```

Comment:
Here and later, I think we can avoid awkward newlines: `Collections.singleton("java.lang...")`